TCAT/Controllers/FavoritesViewController.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  FavoritesViewController.swift
+//  TCAT
+//
+//  Created by Lucy Xu on 11/4/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import Pulley
+import UIKit
+
+class FavoritesViewController: UIViewController {
+
+    // MARK: - View vars
+    private var editButton: UIButton!
+    private let favoritesBlueColor = UIColor(hex: "08A0E0")
+    private var favoritesCollectionView: UICollectionView!
+    private var favoritesTitleLabel: UILabel!
+    private let tab = UIView()
+    private let tabSize = CGSize(width: 32, height: 4)
+
+    // MARK: - Data vars
+    private let editTitle = "Edit"
+    private let favoritesTitle = "Favorites"
```

Comment:
U can just directly call these strings in the setup functions. No need to store this as a variable since its only used once